[2023-07-23T20:27:29.682+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_load_university_data.loading_raw manual__2023-07-23T20:27:19.368671+00:00 [queued]>
[2023-07-23T20:27:29.698+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_load_university_data.loading_raw manual__2023-07-23T20:27:19.368671+00:00 [queued]>
[2023-07-23T20:27:29.698+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-07-23T20:27:29.716+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): loading_raw> on 2023-07-23 20:27:19.368671+00:00
[2023-07-23T20:27:29.759+0000] {standard_task_runner.py:57} INFO - Started process 2238 to run task
[2023-07-23T20:27:29.763+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'daily_load_university_data', 'loading_raw', 'manual__2023-07-23T20:27:19.368671+00:00', '--job-id', '324', '--raw', '--subdir', 'DAGS_FOLDER/LoadInitiator.py', '--cfg-path', '/tmp/tmpspxe7yu3']
[2023-07-23T20:27:29.766+0000] {standard_task_runner.py:85} INFO - Job 324: Subtask loading_raw
[2023-07-23T20:27:29.840+0000] {task_command.py:410} INFO - Running <TaskInstance: daily_load_university_data.loading_raw manual__2023-07-23T20:27:19.368671+00:00 [running]> on host 3906890fc071
[2023-07-23T20:27:30.026+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='daily_load_university_data' AIRFLOW_CTX_TASK_ID='loading_raw' AIRFLOW_CTX_EXECUTION_DATE='2023-07-23T20:27:19.368671+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-23T20:27:19.368671+00:00'
[2023-07-23T20:27:30.029+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-23T20:27:30.030+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/dags/ELTProcess.py']
[2023-07-23T20:27:30.051+0000] {subprocess.py:86} INFO - Output:
[2023-07-23T20:27:31.731+0000] {subprocess.py:93} INFO - WARNING:root:Connection established to Postgres successfully!
[2023-07-23T20:27:31.838+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-23T20:27:31.867+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=daily_load_university_data, task_id=loading_raw, execution_date=20230723T202719, start_date=20230723T202729, end_date=20230723T202731
[2023-07-23T20:27:31.905+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-23T20:27:31.933+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
